pyteal/ast/opup.py

```diff
@@ -87,41 +106,59 @@ def __init__(self, mode: OpUpMode, target_app_id: Expr = None):
 
         self.mode = mode
 
-    def _construct_itxn(self) -> Expr:
+    def _construct_itxn(self, inner_fee: Expr = None) -> Expr:
```

Comment:
@barnjamin Minor:  Are you comfortable with the following _suggestion_?

With the _suggestion_, mypy prevents an unintended regression where the caller fails to supply a `inner_fee` argument.

If you strongly prefer as is, then feel welcomed to resolve without change.

```suggestion
    def _construct_itxn(self, inner_fee: Optional[Expr]) -> Expr:
```